test/unit/errors.test.ts

```diff
@@ -61,9 +61,9 @@ describe('errors', () => {
     // Needed because of https://github.com/Microsoft/TypeScript/wiki/Breaking-Changes#extending-built-ins-like-error-array-and-map-may-no-longer-work
     it('CheckCancelledByClientError should support instanceof', () => {
       const error = new CheckCancelledByClientError({
-        detail: 'The check was cancelled. No result is available.',
-        type: ErrorType.CheckCancelled,
-        title: 'Check cancelled',
+        detail: 'The check was canceled. No result is available.',
+        type: ErrorType.CheckCanceled,
```

Comment:
Syntax Error: Property 'CheckCanceled' does not exist on type 'typeof ErrorType'. Did you mean 'CheckCancelled'?ts(2551)
errors.ts(36, 3): 'CheckCancelled' is declared here.

Reply:
I have made the change